movies/1940/the-great-dictator.json

```diff
@@ -33,5 +32,6 @@
     "Nita Pike",
     "George Lynn"
   ],
-  "storyline": "While the Dictator Adenoid Hynkel attempts to expand his regime. a poor Jewish Barber avoid persecution."
+  "storyline": "During the last days of World War I, a soldier saves the life of a pilot. While fleeing the enemy they are involved in a crash and the soldier loses his memory. After being released from the hospital he opens a barber shop in a Jewish Ghetto. He bear a strong resemblance the the dictator of his country of Tomania, Adenoid Hynkel. His story is followed through arrests, mishaps, and being sent to a concentration camp. Strong allegory to World War II which was underway when the movie was produced."
```

Comment:
How about using this... 
"Dictator Adenoid Hynkel tries to expand his empire while a poor Jewish barber tries to avoid persecution from Hynkel's regime."